dspace-server-webapp/src/test/java/org/dspace/app/rest/RelationshipDeleteRestRepositoryIT.java

```diff
@@ -0,0 +1,574 @@
+    }
+
+    @Test
+    public void testDeleteAuthorRelationshipCopyToBothItems() throws Exception {
+        initPublicationAuthor();
+
+        getClient(adminAuthToken).perform(
+            delete("/api/core/relationships/" + relationship.getID() + "?copyVirtualMetadata=all"))
+            .andExpect(status().isNoContent());
+
+        leftItem = itemService.find(context, leftItem.getID());
+        List<MetadataValue> authorList = itemService.getMetadata(leftItem, "dc", "contributor", "author", Item.ANY);
+        assertThat(authorList.size(), equalTo(1));
+        assertThat(authorList.get(0).getValue(), equalTo("familyName, firstName"));
+        assertThat(authorList.get(0).getMetadataField().getMetadataSchema().getName(), equalTo("dc"));
+        assertThat(authorList.get(0).getMetadataField().getElement(), equalTo("contributor"));
+        assertThat(authorList.get(0).getMetadataField().getQualifier(), equalTo("author"));
+        assertNull(authorList.get(0).getAuthority());
+
+        List<MetadataValue> relationshipMetadataList = itemService
+            .getMetadata(leftItem, "relation", "isAuthorOfPublication", null, Item.ANY);
+        assertThat(relationshipMetadataList.size(), equalTo(0));
+    }
+
+    @Test
+    public void testDeleteJournalRelationshipCopyToLeftItem() throws Exception {
+        initJournalVolumeIssue();
+
+        getClient(adminAuthToken).perform(
+            delete("/api/core/relationships/" + relationship.getID() + "?copyVirtualMetadata=left"))
+            .andExpect(status().isNoContent());
+
+        leftItem = itemService.find(context, leftItem.getID());
+        List<MetadataValue> volumeList =
+            itemService.getMetadata(leftItem, "publicationvolume", "volumeNumber", null, Item.ANY);
+        assertThat(volumeList.size(), equalTo(1));
+        assertThat(volumeList.get(0).getValue(), equalTo("30"));
+
+        rightItem = itemService.find(context, rightItem.getID());
+        List<MetadataValue> issueList =
+            itemService.getMetadata(rightItem, "publicationissue", "issueNumber", null, Item.ANY);
+        assertThat(issueList.size(), equalTo(0));
+    }
+
+    @Test
+    public void testJournalDeleteRelationshipCopyToRightItem() throws Exception {
+        initJournalVolumeIssue();
+
+        getClient(adminAuthToken).perform(
+            delete("/api/core/relationships/" + relationship.getID() + "?copyVirtualMetadata=right"))
+            .andExpect(status().isNoContent());
+
+        leftItem = itemService.find(context, leftItem.getID());
+        List<MetadataValue> volumeList =
+            itemService.getMetadata(leftItem, "publicationvolume", "volumeNumber", null, Item.ANY);
+        assertThat(volumeList.size(), equalTo(0));
+
+        rightItem = itemService.find(context, rightItem.getID());
+        List<MetadataValue> issueList =
+            itemService.getMetadata(rightItem, "publicationissue", "issueNumber", null, Item.ANY);
```

Comment:
This test looks odd...It's testing for the volumeNumber on the LeftItem, but the issueNumber on the RightItem.  Why isn't it testing the same field on both items?

Reply:
These are the virtual metadata fields. This is a bi-directional virtual metadata setup.